packages/online-editor/src/importFromUrl/ImportFromUrlHomePageCard.tsx

```diff
@@ -42,35 +43,56 @@ export function ImportFromUrlCard() {
 
   const [authSessionId, setAuthSessionId] = useState<string | undefined>(AUTH_SESSION_NONE.id);
   const [url, setUrl] = useState("");
+  const [insecurelyDisableTlsCertificateValidation, setInsecurelyDisableTlsCertificateValidation] = useState(false);
   const [gitRefName, setGitRef] = useState("");
 
   const advancedImportModalRef = useRef<AdvancedImportModalRef>(null);
 
   const { authInfo, authSession } = useAuthSession(authSessionId);
 
   const importableUrl = useImportableUrl(url);
-  const clonableUrl = useClonableUrl(url, authInfo, gitRefName);
+
+  const clonableUrl = useClonableUrl(url, authInfo, gitRefName, insecurelyDisableTlsCertificateValidation);
 
   // Select authSession based on the importableUrl domain (begin)
   const authProviders = useAuthProviders();
   const { authSessions, authSessionStatus } = useAuthSessions();
 
+  const updateInsecurelyDisableTlsCertificateValidation = useCallback(
+    (newAuthSession: AuthSession) => {
+      if (newAuthSession && newAuthSession.type === "git") {
+        const localAuthProvider = authProviders.find((provider) => provider.id === newAuthSession.authProviderId);
+        if (localAuthProvider && localAuthProvider.group === AuthProviderGroup.GIT) {
+          setInsecurelyDisableTlsCertificateValidation(
+            localAuthProvider.insecurelyDisableTlsCertificateValidation ?? false
+          );
+        }
+      }
```

Comment:
```suggestion
      if (newAuthSession?.type === "git") {
        const localAuthProvider = authProviders.find((provider) => provider.id === newAuthSession.authProviderId);
        if (localAuthProvider?.group === AuthProviderGroup.GIT) {
          setInsecurelyDisableTlsCertificateValidation(
            localAuthProvider.insecurelyDisableTlsCertificateValidation ?? false
          );
        }
      }
```

^ Isn't it the same? It looks cleaner to me...